backend/tracim_backend/views/collaborative_document_edition_api/wopi_api/wopi_schema.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+import typing
+
+import marshmallow
+from marshmallow import post_load
+
+
+class WopiPutHeadersSchema(marshmallow.Schema):
+    wopi_lool_timestamp = marshmallow.fields.DateTime(
+        required=False, load_from="X-LOOL-WOPI-Timestamp", dump_to="X-LOOL-WOPI-Timestamp"
+    )
+
+    @post_load
+    def make_wopi_put_headers(self, data: typing.Dict[str, typing.Any]) -> object:
```

Comment:
description of content of data in a docstring? (maybe an example)

Reply:
same question as https://github.com/tracim/tracim/pull/2139#discussion_r310031068